### DISCUSSION TITLE
uniqueness-constraint-between-two-columns-in-rails/

### DISCUSSION THREAD

**Discussion by giscus** [nelsonfigueroa/nelson.cloud#20] [Announcements]
# uniqueness-constraint-between-two-columns-in-rails/

Add a uniqueness constraint between two columns in Ruby on Rails.

https://nelson.cloud/uniqueness-constraint-between-two-columns-in-rails/

**Comment by FunnyDrew** [comment]
Thank you, for post. solve my problem with uniqueness.
It's cool remark: Organic content!!!
